README.md

```diff
@@ -1,24 +1,26 @@
+![](docs/figures/overview/logo.png)
+
 # Welcome to the Monty repository!
 *An open-source, sensorimotor learning system following the principles of the neocortex.*
 
-This repository contains the first implementation of a **sensorimotor learning system**
-from the **Thousand Brains Project at Numenta**. We lovingly call it **Monty** after Vernon Mountcastle,
-who proposed cortical columns as a repeating functional unit across the neocortex.
+[![](https://github.com/thousandbrainsproject/tbp.monty/actions/workflows/monty.yml/badge.svg)](https://github.com/thousandbrainsproject/tbp.monty/actions/workflows/monty.yml)
+
+This repository contains the first implementation of a **sensorimotor learning system** from the **Thousand Brains Project at Numenta**. We lovingly call it **Monty** after Vernon Mountcastle, who proposed cortical columns as a repeating functional unit across the neocortex.
 
 This is an open-source project by Numenta, partially funded by the Gates Foundation.
 
-Please find our **full documentation** [here](https://thousandbrainsproject.readme.io/)
+Please find our [**full documentation** here](https://thousandbrainsproject.readme.io/)
+
+Our [**API documentation** here](https://api-monty.thousandbrains.org).
 
 # Getting Started
 
-You can find detailed instructions on how to install the project requirements and how
-to get started [here](https://thousandbrainsproject.readme.io/docs/getting-started)
+You can find detailed instructions on how to install the project requirements and how to get started [here](https://thousandbrainsproject.readme.io/docs/getting-started)
 
 # Current Performance
-We regularly evaluate this system against a set of sensorimotor tasks, summarized in the **[benchmark experiments](./benchmarks/configs/)**. Any time a functional change is made to the code, these experiments are rerun, and results are updated.
+We regularly evaluate this system against a set of sensorimotor tasks, summarized in the [benchmark experiments](./benchmarks/configs/). Any time a functional change is made to the code, these experiments are rerun, and results are updated.
```

Comment:
What's the reason you unbolded this?

Reply:
It seemed inconsistent.  But then I added a bolded link in my changes.  🤷 :)  I'll revert.

Reply:
okay yeah I don't have a strong opinion on this. You can have it however you think it looks better in the rendered version. Was just curious if there was something that didn't work with bolding a link.